### PR TITLE
ITS Track: V0s Invariant Mass histograms added

### PR DESCRIPTION
**Pull request by eeremenk**

### CODE REVIEW

File: Modules/ITS/src/ITSTrackTask.cxx [hunk 1]
```diff
@@ -177,6 +179,22 @@ void ITSTrackTask::monitorData(o2::framework::ProcessingContext& ctx)
...
+  using Vec3D = ROOT::Math::SVector<double, 3>; // this is a type of the fitted vertex
+  o2::vertexing::DCAFitter2 ft;
+  ft.setBz(-5.0);
+  ft.setPropagateToPCA(true);
+  ft.setMaxR(30);
+  ft.setMaxDZIni(0.1);
+  ft.setMaxDXYIni(0.1);
+  ft.setMinParamChange(1e-3);
+  ft.setMinRelChi2Change(0.9);
+  ft.setMaxChi2(10);
```
Comment (IsakovAD) [inline Modules/ITS/src/ITSTrackTask.cxx:192]: Is it possible to add here comments about meaning of parameters you are using? It is not crucial, but would be very useful for code support

**Review by IsakovAD — APPROVED** [review]
All is fine

**Comment by IsakovAD** [comment]
@Barthelemy @knopers8 

Could you please add your approval to this PR? We need to have one more review, but Ivan is out of office for this week

**Review by knopers8 — APPROVED** [review]

**Comment by knopers8** [comment]
The PR specifically waits for Ivan's review. I will force merge as soon as I see the macOS check go green.

**Comment by IsakovAD** [comment]
> The PR specifically waits for Ivan's review. I will force merge as soon as I see the macOS check go green.

@knopers8 Ok, understood. How can I initialize the MacOS check? It waits for status to reported for few days

**Comment by knopers8** [comment]
It turns out there is a central issue with the CI on mac. We're investigating.

**Pull request merged**